viite-UI/src/model/SelectedLinkProperty.js

```diff
@@ -339,11 +339,9 @@
                 return _.some(foundDuplicatedLink);
             });
             previousAdjacents = filteredAdjacents;
-            var unknownAdjacents = _.reject(filteredAdjacents, function(t){
-               return t.roadLinkType === RoadLinkType.FloatingRoadLinkType.value;
-            });
+
             var markedRoads = {
-              "adjacents": _.map(applicationModel.selectionTypeIs(selectionType.Floating) ? unknownAdjacents : unknownAdjacents, function (a, index) {
+              "adjacents": _.map(applicationModel.selectionTypeIs(selectionType.Floating) ? filteredAdjacents : filteredAdjacents, function (a, index) {
```

Comment:
Why there is conditional (ternary) operator here? It same as 
`if(something)
  value
else
  value`
which doesn't make any sense since it chooses filteredAdjacents anyway. Or am I missing something?